posts/mobs-are-not-always-destructive.md

```diff
@@ -0,0 +1,94 @@
+
+## Balancing Switch Time (N)
+
+One common aspect we have changed in the various sessions we have held is the switch time. We typically start by asking the group what duration everyone feels would be best for the piece of work we are looking to complete. But an outside observer might wonder two things:
+
+1. How does anybody come up with a seemly arbitrary length of time?
+1. How does the group arrive at a consensus?
+
+The most straightforward way our team has arrived at a switch time is by considering the amount of work involved. If this is a feature epic or more nebulous piece of work with a loftier goal, our switch time will be set accordingly high. We have also used story points to decide on switch times. 1 and 2 point stories have not been considered for mobbing on our team, so it is a pretty easy 1:1 correlation of X story points to N minutes.
+
+The next factor our team uses for determining switch time is the number of people in the mob. A group of three or four developers will require each developer to drive more frequently than one of seven. Now, because our team tends to get easily distracted in general, we need to combat short attention spans while not changing drivers so frequently that it hinders the integrity of the solution. We want to find the right balance of switch time so we can keep the group motivated and engaged.
+
+Last major piece of criteria is the allotted time. For us it has been anywhere from a half hour to a whole, six-hour day. But why this matters is because, just as the agricultural revolution found, it is important to rotate break times into your product's growth. Either halfway in or after each person has gone through a full rotation, we introduce a dedicated break to give people a breather. (This is separate from the guideline which advises stepping away when one is forced to do something else outside of the mobbing topic.)
+
+Say we are mobbing on a feature epic with eight people for an allotted time of five hours, our switch time will probably sit at around 15 minutes. In comparison, a smaller mobbing session of 45 minutes on a three-point story with four developers have had switch times of only five minutes. So why no formula to determining switch time (N)? Someone could propose one, however, this is analogous with story pointing which has no perfect formula either.
+
+## Past Mobbing Goals
+
+For our initial mobbing, the primary objective was to: "Implement First A/B Test". Our A/B Test would be to provide all users with either Experience A or Experience B in the application. With a fifty-fifty split between showing A or B, our team was to implement functionality where each user will be shown educational content in different locations on the page.
+
+Subsequent mobbing goals have included implementations of:
```

Comment:
One thing we need to be careful about is how specific we are calling out features we've worked on.  We may need to abstract this up a level and make it more general somehow.

Reply:
I did try to be general. And I would be open to suggestions for making it more abstracted.